tests/FilesHooksTest.php

```diff
@@ -284,13 +332,55 @@ public function testAddNotificationsForFileAction($filterUsers, $addNotification
 			->willReturn([
 				'ownerPath' => '/owner/path',
 				'users' => [
-					'user' => '/user/path',
-					'user1' => '/user1/path',
-					'user2' => '/user2/path',
+					'user' => '/user/files/path',
+					'user1' => '/user1/files/path',
+					'user2' => '/user2/files/path',
 				],
 				'remotes' => [],
 			]);
 
+		$this->config->expects($this->once())
+			->method('getSystemValueBool')
+			->with('activity_use_cached_mountpoints', false)
+			->willReturn($mountCacheUsed);
```

Comment:
Undefined variable: mountCacheUsed

You added it in the doc block, but not the arguments list